pkg/pool-utils/test/factories/BasePoolFactory.test.ts

```diff
@@ -108,4 +113,34 @@ describe('BasePoolFactory', function () {
       });
     });
   });
+
+  describe('with invalid pause durations', () => {
```

Comment:
You're not testing that the durations are correctly passed all the way down to the factory wide, only that invalid ones are rejected. We don't need to test the full behavior, but we do have to test that the usage of that module is correct.

(this is one of the tests that would happen in the make-base-pool-factory-pausable pr)